cmd/src/bin/tikv-ctl.rs

```diff
@@ -1030,6 +1030,18 @@ fn warning_prompt(message: &str) -> bool {
 }
 
 fn main() {
+    // The config is only used to initialize the global logger.
+    let rocksdb_info_dir = tempfile::tempdir().unwrap();
+    let raftdb_info_dir = tempfile::tempdir().unwrap();
+    #[allow(clippy::field_reassign_with_default)]
+    {
+        let mut cfg = TiKvConfig::default();
+        cfg.log_file = "./tikv-ctl.log".to_owned();
```

Comment:
Can the log file be configurable instead of hard coded? Also it changed tikv-ctl's behavior, a user familiar with the old version may find that a log file shows up in the working directory unexpectedly.
I suggest it do not write the log file by default, and specify the log file if it's needed.